### ISSUE TITLE
Does anyone have a Bot winning rate 60%+ For 24/7 trade

### ISSUE THREAD

**Issue by freedayz04** [binary-com/binary-bot#2386]
Does anyone have a Bot winning rate 60%+ For 24/7 trade

**Comment by sohailsb** [comment]
Yes i have.
if you want test Whatsapp +923007860567

**Comment by ZDEGLET** [comment]
24\7 это шляпа

**Comment by hotmatrixx** [comment]
> Does anyone have a Bot winning rate 60%+ For 24/7 trade

That's easy.
Just run a Synthetic Digit Differs strategy.

All Synthetics run 24/7, and are the only type of trade that does.
Digit Differs has a default win rate of 90%, even if you randomly pick every trade. However, it has a much, much lower payout.

You're new. Winrates in Binary Options are a fallacy. It depends on the type of trade, payout, and your Money Management Strategy.

Those of us who are profitable do not give our code away, we are only here to help new players learn and improve their code.

Anyone trying to sell you bots is probably not making money on their bots, because a winning strategy is more valuable than any amount of money. If it was a winner, they wouldnt need to sell it. They also don't want to give it away, because if everyone follows their strategy, it doesn't work anymore.

Also, coming here and instantly asking someone to share all their hard work and potential earnings without ever having had something to offer in return, can be seen as rude.

Welcome to bot.binary.com

Don't let anyone scam you.

**Comment by ceakuk** [comment]
@hotmatrixx 
your way of calculating win rate is wrong
your calculation : `100*number_of_wins/number_of_trades`
good calculation:`100*sum(profits)/(sum(profits) + sum(abs(losses)))`

**Comment by Martinez222** [comment]
It's easy to say "Those of us who are profitable".Maybe you work for this broker.
Why do you encourage us to try? Just because you are "profitable"?If u want to realy try u have to go live and than  the equation changes .  There are many around here who are ready to "help" you, but do they really want to do this or do they want to make you lose money?
In the last 2 years I have tried many bots, first i bought them, then i made it my self. The conclusion is simple you can not beat their code behind their supercomputer.(i'm talking about Volatility markets).Even so IF u manage to make a good strategie and go live u will have unexpected interruptions, jams, mifid II,broker rulles  etc.

**Comment by hotmatrixx** [comment]
> 
> 
> @hotmatrixx
> your way of calculating win rate is wrong
> your calculation : `100*number_of_wins/number_of_trades`
> good calculation:`100*sum(profits)/sum(amount_traded)`

I know. That was my whole point. he was asking for the wrong thing. I even said that 'however it has a much, much lower payout'. 

he's obviously new.
I do understand that, and I was pointing out that he needs to change his thinking and increase his understanding.

also, my method of calculating winrate was his method, not mine, and I was showing his was flawed - you did describe the same thing I was getting at, albiet mathematically. We were making the same point, just in a different way.

Thanks tho.

**Comment by ceakuk** [comment]
> 
> 
> It's easy to say "Those of us who are profitable".Maybe you work for this broker.
> Why do you encourage us to try? Just because you are "profitable"?If u want to realy try u have to go live and than the equation changes . There are many around here who are ready to "help" you, but do they really want to do this or do they want to make you lose money?
> In the last 2 years I have tried many bots, first i bought them, then i made it my self. The conclusion is simple you can not beat their code behind their supercomputer.(i'm talking about Volatility markets).Even so IF u manage to make a good strategie and go live u will have unexpected interruptions, jams, mifid II,broker rulles etc.
I don't agree with you that you can't beat there  supercomputer because if the following reasons
**DIGITS:**
these are random numbers. you loose on digits because they are random numbers not because of manipulation. the fact that you have lost a couple of times does not make it likely that you will win soon. say you have been loosing and by 5th trade you recouver, what make it not 6th next time, what about 7th, .......... soon you will need to recouver at 20th......100th........1000....10000 trades. you will sooner or later blow your account. as you increase your bets, the loss becomes exponential

**NON** DIGITS VOLATILITY 
just as digits, the   the signal is made in a way that it can be quiet any time or explode any time for unknown  duration. 
overbought underbought is useless because the signal is not dependent on the number of buyers against sellers

**Comment by hotmatrixx** [comment]
> 
> 
> It's easy to say "Those of us who are profitable".Maybe you work for this broker.
> Why do you encourage us to try? Just because you are "profitable"?If u want to realy try u have to go live and than the equation changes . There are many around here who are ready to "help" you, but do they really want to do this or do they want to make you lose money?
> In the last 2 years I have tried many bots, first i bought them, then i made it my self. The conclusion is simple you can not beat their code behind their supercomputer.(i'm talking about Volatility markets).Even so IF u manage to make a good strategie and go live u will have unexpected interruptions, jams, mifid II,broker rulles etc.

look, I understand where you are coming from.
No, I have no association with the broker. don't work for em, not paid, no commissions.. nothing. I'm just a random that uses the bot like everyone else, I've just been using it a lot longer.

first of all, if you started by buying bots, then I hope you learned your lesson. NO ONE sells profitable code. if it's profitable, it's worth more kept secret.

second. I said, Synthetic markets. Read some of my other threads here. I hate on them regularly, but they are good for doing quick tests on VTR accounts, to make sure that things like your martingale works, or something. But NEVER put live money on Synthetics. That's a scam. I ONLY trade Forex Major - the results have to be real, and the broker is accountable if the spread differs more than their contracts allow. The equation isn't allowed to change on those markets.

third. I've been playing with binary options for 3 years, (almost) every day.  I've been algo (bot) trading for around a year. I'm making money. real money, but it's slow, and my account is low so I'm really cautious and I pick my timing very carefully. I don't make enough to make a full time living, but when I run my bot, my sessions are profitable.

TLDR: I agree with you. 

Why do I encourage us to 'try'.
I didn't. I don't. I said I would help people learn to code. I never recommended it. In fact, most of my post was warnings, not encouragement.

Although in saying that, for some reason I didn't put 'I don't trust Synthetic markets' in my post, which I usually do, if you look at my other threads. You mentioned 'volitility' markets, we basically meant the same thing, but Synthetics is more encompassing. I don't like saying they're a scam, but I personally won't trust an index that is literally "anything the broker wants to make up". That seems like a conflict of interest to me.

**Comment by hotmatrixx** [comment]
> 
> 
> > It's easy to say "Those of us who are profitable".Maybe you work for this broker.
> > Why do you encourage us to try? Just because you are "profitable"?If u want to realy try u have to go live and than the equation changes . There are many around here who are ready to "help" you, but do they really want to do this or do they want to make you lose money?
> > In the last 2 years I have tried many bots, first i bought them, then i made it my self. The conclusion is simple you can not beat their code behind their supercomputer.(i'm talking about Volatility markets).Even so IF u manage to make a good strategie and go live u will have unexpected interruptions, jams, mifid II,broker rulles etc.
> > I don't agree with you that you can't beat there  supercomputer because if the following reasons
> 
> **DIGITS:**
> these are random numbers. you loose on digits because they are random numbers not because of manipulation. the fact that you have lost a couple of times does not make it likely that you will win soon. say you have been loosing and by 5th trade you recouver, what make it not 6th next time, what about 7th, .......... soon you will need to recouver at 20th......100th........1000....10000 trades. you will sooner or later blow your account. as you increase your bets, the loss becomes .
> 
> **NONDIGITS-VOLATILITY:**
> just as digits, the the signal is made in a way that it can be quiet any time or explode any time for unknown duration.
> overbought underbought is useless because the signal is not dependent on the number of buyers against sellers

yeah. interestingly I made a bot a while back that counted digits and placed trades based on how many of each type there were. after 10,000 runs there were almost exactly 1000 of each digit. I found that interesting.

if the 'player' lost to the digits market, it's due to any number of things, but the market is missing a vital factor that can be manipulated; human emotion. That's what money is made on, not 'beating the house' which is what most people come here to try and do.

quick note; your losses are only exponential, if you're using an exponential money management system, and with digi markets you have to because the win/loss rates are perfectly spread, otherwise you'll just lose at a rate of half the negative inverse of the payout. ie, around 2% per trade overall.

**Comment by ceakuk** [comment]
> 
> 
> > It's easy to say "Those of us who are profitable".Maybe you work for this broker.
> > Why do you encourage us to try? Just because you are "profitable"?If u want to realy try u have to go live and than the equation changes . There are many around here who are ready to "help" you, but do they really want to do this or do they want to make you lose money?
> > In the last 2 years I have tried many bots, first i bought them, then i made it my self. The conclusion is simple you can not beat their code behind their supercomputer.(i'm talking about Volatility markets).Even so IF u manage to make a good strategie and go live u will have unexpected interruptions, jams, mifid II,broker rulles etc.
> 
> look, I understand where you are coming from.
> No, I have no association with the broker. don't work for em, not paid, no commissions.. nothing. I'm just a random that uses the bot like everyone else, I've just been using it a lot longer.
> 
> first of all, if you started by buying bots, then I hope you learned your lesson. NO ONE sells profitable code. if it's profitable, it's worth more kept secret.
> 
> second. I said, Synthetic markets. Read some of my other threads here. I hate on them regularly, but they are good for doing quick tests on VTR accounts, to make sure that things like your martingale works, or something. But NEVER put live money on Synthetics. That's a scam. I ONLY trade Forex Major - the results have to be real, and the broker is accountable if the spread differs more than their contracts allow. The equation isn't allowed to change on those markets.
> 
> third. I've been playing with binary options for 3 years, (almost) every day. I've been algo (bot) trading for around a year. I'm making money. real money, but it's slow, and my account is low so I'm really cautious and I pick my timing very carefully. I don't make enough to make a full time living, but when I run my bot, my sessions are profitable.
> 
> TLDR: I agree with you.
> 
> Why do I encourage us to 'try'.
> I didn't. I don't. I said I would help people learn to code. I never recommended it. In fact, most of my post was warnings, not encouragement.
> 
> Although in saying that, for some reason I didn't put 'I don't trust Synthetic markets' in my post, which I usually do, if you look at my other threads. You mentioned 'volitility' markets, we basically meant the same thing, but Synthetics is more encompassing. I don't like saying they're a scam, but I personally won't trust an index that is literally "anything the broker wants to make up". That seems like a conflict of interest to me.

you nailed it in every aspect.
you can never win in the long run on anthing where return_on_investment* probability_of_wining<1
example digitmatch =8.09*(1/10)=0.809
with forex, ou can win slow but steady

**Comment by ceakuk** [comment]
> 
> 
> > > It's easy to say "Those of us who are profitable".Maybe you work for this broker.
> > > Why do you encourage us to try? Just because you are "profitable"?If u want to realy try u have to go live and than the equation changes . There are many around here who are ready to "help" you, but do they really want to do this or do they want to make you lose money?
> > > In the last 2 years I have tried many bots, first i bought them, then i made it my self. The conclusion is simple you can not beat their code behind their supercomputer.(i'm talking about Volatility markets).Even so IF u manage to make a good strategie and go live u will have unexpected interruptions, jams, mifid II,broker rulles etc.
> > > I don't agree with you that you can't beat there  supercomputer because if the following reasons
> > 
> > 
> > **DIGITS:**
> > these are random numbers. you loose on digits because they are random numbers not because of manipulation. the fact that you have lost a couple of times does not make it likely that you will win soon. say you have been loosing and by 5th trade you recouver, what make it not 6th next time, what about 7th, .......... soon you will need to recouver at 20th......100th........1000....10000 trades. you will sooner or later blow your account. as you increase your bets, the loss becomes .
> > **NONDIGITS-VOLATILITY:**
> > just as digits, the the signal is made in a way that it can be quiet any time or explode any time for unknown duration.
> > overbought underbought is useless because the signal is not dependent on the number of buyers against sellers
> 
> yeah. interestingly I made a bot a while back that counted digits and placed trades based on how many of each type there were. after 10,000 runs there were almost exactly 1000 of each digit. I found that interesting.
> 
> if the 'player' lost to the digits market, it's due to any number of things, but the market is missing a vital factor that can be manipulated; human emotion. That's what money is made on, not 'beating the house' which is what most people come here to try and do.
> 
> quick note; your losses are only exponential, if you're using an exponential money management system, and with digi markets you have to because the win/loss rates are perfectly spread, otherwise you'll just lose at a rate of half the negative inverse of the payout. ie, around 2% per trade overall.

A real life example of trading digits is making babies. some get boys, some get girls and some get both. if you have just boys, trying more can give you girls but can also give you more boys. its random numbers like volatility index. summing all sexes in the world  give 50%+-5% boys or girls which is statistically 50% each.  summing all RISE or FALL can be 50% by luck in the short time but in the long run will be 50% . long run can be 100,1000, 1million, there is no limit

if  even the world population of over a billion, w

**Comment by muroyal** [comment]
https://github.com/binary-com/binary-bot/issues/617

**Comment by MBarros91** [comment]
I'm trying to build one 
https://github.com/binary-com/binary-bot/issues/3221

**Comment by Sonal-sl** [comment]
Hey I am from Sri lanka please send me a binary.com bot no consecutive double losses bot to my E mail
sonalnimsara229@gmail.com
